code/src/components/MoviePage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState} from 'react';
+import {  useParams } from 'react-router-dom';
+
+import './moviePage.css'
+
+export const MoviePage = () => {
+    const { id } = useParams();
+    const [movie, setMovie] = useState({});
+
+    useEffect(() => {
+        fetch(`https://api.themoviedb.org/3/movie/${id}?api_key=0cb608022c0a0fbef75881f68133d694&language=en-US`)
+        .then((res) => res.json())
+        .then((json) => {
+            setMovie(json)
+        });
+    }, []);
+
+ 
+    return (
+        
+        <section className='movie-page-background' 
+        style={{backgroundImage:`linear-gradient(rgba(0, 0, 0, 0) 70%, rgb(0, 0, 0) 100%), url(https://image.tmdb.org/t/p/w1280${movie.backdrop_path})`}}>
+
+        
+        <div className='movie-page-container'>
+          <img src={`https://image.tmdb.org/t/p/w342${movie.poster_path}`} alt={movie.title}/>
```

Comment:
If you want to add a link back to the MovieList page the you can import Link from the react router dom at the top of the page
`import { Link } from 'react-router-dom';`
and then write the link like this in your return
```
<Link to="/">
             Back to film list
</Link>
```